browser/net/brave_block_reporting_urls_unittest.cc

```diff
@@ -0,0 +1,91 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/net/brave_block_reporting_urls.h"
+
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "brave/browser/net/url_context.h"
+#include "chrome/test/base/chrome_render_view_host_test_harness.h"
+#include "net/traffic_annotation/network_traffic_annotation_test_helper.h"
+#include "net/url_request/url_request_test_util.h"
+#include "url/gurl.h"
+#include "url/url_constants.h"
+
+namespace {
+
+const char kInvalidUrl[] = "https://no-thanks.invalid";
+
+const std::vector<const std::string> kNormalUrls({
```

Comment:
cc prohibits non-trivial global values, maybe just rewrite it using `const char*`?

Reply:
or transform these arrays to local vars

Reply:
Converted to local vars.